targetingpacks.props

```diff
@@ -8,7 +8,7 @@
   <PropertyGroup>
     <ContractOutputPath>$(BinDir)netcoreapp/TargetingPack</ContractOutputPath>
     <FrameworkPathOverride>$(ContractOutputPath)</FrameworkPathOverride>
-    <AssemblySearchPaths>$(AssemblySearchPaths);$(ContractOutputPath)</AssemblySearchPaths>
+    <AssemblySearchPaths>$(AssemblySearchPaths);$(ContractOutputPath);{RawFileName}</AssemblySearchPaths>
```

Comment:
The `Microsoft.TargetingPack.Private.CoreCLR` and `Microsoft.TargetingPack.Private.WinRT` depproj's are both still using project.json's to build, and they were giving build warnings without this.